src/Listener/MonitoringTokenListener.php

```diff
@@ -31,15 +32,15 @@ public function __construct (HostingConfig $config)
     public function onResponse (FilterResponseEvent $event) : void
     {
         // skip if not master request
-        if (!$event->isMasterRequest() || null === $this->uptimeHtmlEmbed)
+        if (null === $this->uptimeHtmlEmbed || !$event->isMasterRequest())
```

Comment:
Please revert.

Reply:
This has been changed as PhpStorm said that it's faster to check for null than running that method. Should we just ignore that?

Reply:
It doesn't really give us anything. You can also leave it.